src/Menu.php

```diff
@@ -93,45 +91,44 @@ public function addHeader($name)
      */
     public function addMenu($name)
     {
-        if (is_array($name)) {
-            $label = $name[0];
-            unset($name[0]);
-        } else {
-            $label = $name;
-            $name = [];
-        }
+        $subMenu = $this->add([new self(), 'defaultTemplate' => 'submenu.html', 'ui' => 'dropdown', 'in_dropdown' => true]);
 
-        $sub_menu = $this->add([new self(), 'defaultTemplate' => 'submenu.html', 'ui' => 'dropdown', 'in_dropdown' => true]);
-        $sub_menu->set('label', $label);
+        $name = (array) $name;
+
+        if ($label = $name['title'] ?? $name['label'] ?? $name['name'] ?? $name[0] ?? '') {
+            $subMenu->set('label', $label);
+        }
 
-        if (isset($name['icon']) && $name['icon']) {
-            $sub_menu->add(new Icon($name['icon']), 'Icon')->removeClass('item');
+        if (!empty($name['icon'])) {
+            $subMenu->add(new Icon($name['icon']), 'Icon')->removeClass('item');
         }
 
         if (!$this->in_dropdown) {
-            $sub_menu->js(true)->dropdown(['on' => 'hover', 'action' => 'hide']);
+            $subMenu->js(true)->dropdown(['on' => 'hover', 'action' => 'hide']);
         }
 
-        return $sub_menu;
+        return $subMenu;
     }
 
     /**
      * Adds menu group.
      *
-     * @param string|array $title
+     * @param string|array $name
      *
      * @return Menu
      */
-    public function addGroup($title)
+    public function addGroup($name)
     {
         $group = $this->add([new self(), 'defaultTemplate' => 'menugroup.html', 'ui' => false]);
-        if (is_string($title)) {
+
+        $name = (array) $name;
+
+        if ($title = $name['title'] ?? $name['label'] ?? $name['name'] ?? $name[0] ?? '') {
```

Comment:
and theoretically $title can be zero or '0'

Reply:
fixed with a6b535b